ccmlib/scylla_node.py

```diff
@@ -310,15 +310,15 @@ def _start_scylla(self, args, marks, update_pid, wait_other_notice,
 
         if wait_other_notice:
             for node, _ in marks:
-                t = timeout if timeout is not None else 120 if self.cluster.scylla_mode != 'debug' else 600
+                t = self.cluster.default_wait_other_notice_timeout
                 node.watch_rest_for_alive(self, timeout=t)
                 self.watch_rest_for_alive(node, timeout=t)
 
         if wait_for_binary_proto:
-            t = timeout * 4 if timeout is not None else 420 if self.cluster.scylla_mode != 'debug' else 900
+            t = self.cluster.default_wait_for_binary_proto
             from_mark = self.mark
             try:
-                self.wait_for_binary_interface(from_mark=from_mark, process=self._process_scylla, timeout=timeout or 180)
```

Comment:
@bhalevy I don't know if it's on purpose that 180 is hard coded here, and the `t` was only used in the exception

Reply:
It's probably a bug.
I don't think we should maintain it

Reply:
I've put the same timeout for all of the calls under `wait_for_binary_proto`

Reply:
ok, but we need to explain the reversal of order here in the commit message and/or code comment

Reply:
@bhalevy updated the commit message with more details as for the order change